gnucash/report/html-utilities.scm

```diff
@@ -413,11 +413,14 @@
    (string->char-set ":/?#[]@")         ;gen-delims
    (string->char-set "-._~")))
 
+;;path must be absolute. On Windows an absolute path begins with a
+;;drive letter followed by a colon.
 (define (make-uri path)
-  (string-append
-   "file://"
-   (uri-encode (gnc:substring-replace path "\\" "/")
+  (let ((uri-path (uri-encode (gnc:substring-replace path "\\" "/")
                #:unescaped-chars unreserved-chars-rfc3986)))
+  (string-append
+   (if (eq? (car (string->list uri-path)) "/") "file://" "file:///")
```

Comment:
`(eq? (car (string->list uri-path)) "/")` to `(char=? (string-ref uri-path 0) #\/)` will avoid creating intermediate list, and we'd want to be careful about equality functions.

https://www.gnu.org/software/guile/manual/html_node/Equality.html

Reply:
OK, done.